### PR TITLE
[FIX] l10n_cn_small_business: Modify Small And Medium-sized Enterpris…

### PR DESCRIPTION
**Pull request by Colinliz**
…es COA

Steps to reproduce the bug:
- Modify some account according to 'Accounting Standards for 2017 Small and Medium-sized Enterprises'
- Add account translation into .po file

Signed-off-by: Colinliz <lizheng02@inspur.com>

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Colinliz** [comment]
hi @lap-odoo ,I have modified the Small And Medium-sized Enterprise COA and submitted the translation file. Please review the code I submitted. Thank you~

**Comment by sle-odoo** [comment]
robodoo retry

**Comment by sle-odoo** [comment]
robodoo retry

File: addons/l10n_cn_small_business/data/account.account.template.csv [hunk 1]
```diff
@@ -1,77 +1,69 @@
...
+"l10n_cn_1702","累计摊销","1702","account.data_account_type_depreciation","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1801","长期待摊销费用","1801","account.data_account_type_depreciation","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_1901","待处理财产损溢","1901","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"l10n_cn_2001","短期借款","2001","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2201","应付票据","2201","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2202","应付账款","2202","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2203","预收账款","2203","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2211","应付职工薪酬","2211","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
+"l10n_cn_2221","应交税费","2221","account.data_account_type_payable","l10n_cn_small_business.l10n_chart_china_small_business","True"
```
Comment (GoodERPJeff) [inline addons/l10n_cn_small_business/data/account.account.template.csv]: Is this account need  reconcile ?

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml [hunk 2]
```diff
@@ -3,19 +3,15 @@
 
 <!--
 
-     Copyright (C) 2012-2012 南京盈通 ccdos@intoerp.com <small business chart>
-     
-会计科目表模板( 小企业会计准则2011)
+Copyright (C) 2019-now PS Cloud pscloud@inspur.com <small business chart>
```
Comment (GoodERPJeff) [inline addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml]: Copyright要新增行，不能去掉原有的行

File: addons/l10n_cn_small_business/data/account.account.template.csv (hunk 1 above)
Comment (lap-odoo) [reply addons/l10n_cn_small_business/data/account.account.template.csv]: @GoodERPJeff Normally, all payable accounts need to allow the reconciliation. Why do you think they shouldn't be in reconciled=true ?

**Pull request closed without merging**